west.yml

```diff
@@ -95,7 +95,7 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 80a337dc4cc980f80e595971147eb41458b29ae8
+      revision: pull/16/head
```

Comment:
Still need to resolve. Please see https://github.com/zephyrproject-rtos/hal_nxp/pull/16#issuecomment-618023931

Reply:
Done!

Reply:
Thanks! The hal PR has been merged, so you can now update this to reference the new SHA.